specs/data_structures.md

```diff
@@ -773,59 +774,54 @@ The state of the LazyLedger chain is intentionally restricted to containing only
 
 The state tree is separated into `2**(8*STATE_SUBTREE_RESERVED_BYTES)` subtrees, each of which can be used to store a different component of the state. This is done by slicing off the highest `STATE_SUBTREE_RESERVED_BYTES` bytes from the key and replacing them with the appropriate [reserved state subtree ID](consensus.md#reserved-state-subtree-ids). Reducing the key size within subtrees also reduces the collision resistance of keys by `8*STATE_SUBTREE_RESERVED_BYTES` bits, but this is not an issue due the number of bits removed being small.
 
-Three subtrees are maintained:
+A number of subtrees are maintained:
 1. [Accounts](#account)
 1. [Active validator set](#validator)
 1. [Inactive validator set](#validator)
+1. [Delegation set](#delegation)
 
 ### Account
 
-| name             | type                      | description                                                                       |
-| ---------------- | ------------------------- | --------------------------------------------------------------------------------- |
-| `balance`        | [Amount](#type-aliases)   | Coin balance.                                                                     |
-| `nonce`          | [Nonce](#type-aliases)    | Account nonce. Every outgoing transaction from this account increments the nonce. |
-| `isValidator`    | `bool`                    | Whether this account is a validator or not.                                       |
-| `isDelegating`   | `bool`                    | Whether this account is delegating its stake or not.                              |
-| `delegationInfo` | [Delegation](#delegation) | _Optional_, only if `isDelegating` is set. Delegation info.                       |
-
-In the accounts subtree, accounts (i.e. leaves) are keyed by the [hash](#hashdigest) of their [address](#address). The first byte is then replaced with `ACCOUNTS_SUBTREE_ID`.
-
-### Delegation
-
 ```C++
-enum DelegationStatus : uint8_t {
-    Bonded = 1,
-    Unbonding = 2,
+enum AccountStatus : uint8_t {
+    None = 1,
```

Comment:
While the other states are more or less self-explanatory, it is not clear what this stands for. Maybe add a comment here?

Reply:
Fixed in 2da3d9b.